packages/components/src/Gallery/Gallery.tsx

```diff
@@ -22,7 +22,11 @@ export function Gallery({ files, size = "base", max, onDelete }: GalleryProps) {
           return (
             <FormatFile
               key={file.key}
-              file={{ ...file, src: () => Promise.resolve(file.src) }}
+              file={{
+                ...file,
+                uploadUrl: "",
```

Comment:
Is there a reason why not just make the `uploadUrl` optional in the type?
`uploadUrl ?: string`

Reply:
Yep it's fair, I was wondering here just because I made a lot of changes in JO to use this prop as required but anyway I can fix that in there later!

Reply:
Thanks, I didn't see it in the original PR but by making it a required prop it was technically a breaking change and I don't think it was noticed at the time.